dspace-api/src/main/java/org/dspace/content/service/ItemService.java

```diff
@@ -37,6 +37,9 @@
  * @author kevinvandevelde at atmire.com
  */
 public interface ItemService extends DSpaceObjectService<Item>, DSpaceObjectLegacySupportService<Item> {
+
+    public Item find(Context context, UUID id) throws SQLException;
```

Comment:
Add javadocs for the 5 new methods in this Class

Reply:
removed unnecessary methods and add javadoc to the residual https://github.com/DSpace/DSpace/pull/2312/files#diff-ce065cd345298b0ebf3a369801eb7399R654